liquibase-core/src/main/java/liquibase/command/core/InternalHistoryCommandStep.java

```diff
@@ -132,12 +164,72 @@ void printReport(PrintWriter output) {
             output.println("");
         }
 
-        String getDeploymentId() {
-            if (changeSets.size() == 0) {
-                return null;
+        @Override
+        public String getDeploymentId() {
+            return changeSets.stream()
+                    .findFirst()
+                    .map(RanChangeSet::getDeploymentId)
+                    .orElse(null);
+        }
+    }
+
+    public static class TabularDeploymentDetails implements ReportPrinter {
+
+        private static final List<String> HEADERS = Arrays.asList(
+                "Deployment ID",
```

Comment:
To make the code consistent with documentation and other output by the CLI, can we update the headers to be:

```
Update Date
Changelog Path
Changeset Author
Changeset ID
```
We recently did a clean up of places where we had `change set` instead of `changeset`, otherwise I wouldn't mention this.